empty/public/phaser/main.js

```diff
@@ -74,14 +78,14 @@ const mainState = {
     // It contains the game's logic
     // If the bird is out of the screen (too high or too low)
     // Call the 'restartGame' function
-    if (sprite.y < 0 || sprite.y > 490)
+    if (sprite.y < 0 || sprite.y > 735)
```

Comment:
Consider declaring all of these magic numbers as constants, with intention revealing names.